Cargo.toml

```diff
@@ -12,7 +12,7 @@ readme = "README.md"
 license = "MIT"
 
 [dependencies]
-bdk = { version = "^0.7", default-features = false, features = ["all-keys"]}
+bdk = { version = "0.10.0", default-features = false, features = ["all-keys"]}
```

Comment:
Good Idea. Better to enable `tls` in `ureq` if we use it. It seems like a general requirement to be able to connect https. Will open a PR if we agree..

Reply:
True, maybe best of have `tls` feature enabled by default in `bdk`, but first I'd like to get @tcharding's input since he originally added the `ureq` dependency and may have an argument for leaving `tls` off by default.

Reply:
I have no problem with enabling TLS by default. As for why I disabled it, I can't remember exactly why, I probably just disabled default features during a debugging session and never turned TLS back on. Thanks for checking in with me and thanks for giving it the thought to realize we should enable it by default!